src/lib/features/segment/segment-store.ts

```diff
@@ -145,6 +144,11 @@ export default class SegmentStore implements ISegmentStore {
                 `${T.featureStrategies}.id`,
                 `${T.featureStrategySegment}.feature_strategy_id`,
             )
+            .leftJoin(
```

Comment:
added a new join to get feature archived info

Reply:
I also tried the where clause just before groupBy instead of rewriting counts but it didn't work.